packages/apps/dashboard/ui-2024/src/features/Leaderboard/components/Table.tsx

```diff
@@ -184,31 +185,36 @@ export const Table = ({
 										},
 									}}
 								>
-									<Grid
-										container
-										wrap="nowrap"
-										alignItems="center"
-										gap="8px"
-										justifyContent="flex-start"
+									<Link
+										to={row.url}
```

Comment:
We should be using website key from kvstore. 
If website is not set it should not be clickable.
It might require some subgraph changes.